src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -20,7 +20,7 @@
   // Main
   --#{$text-input-group}__main--first-child--not--text-input--MarginInlineStart: calc(var(--pf-t--global--spacer--control--horizontal--plain) / 2);
   --#{$text-input-group}__main--m-icon__text-input--PaddingInlineStart: calc(var(--pf-t--global--spacer--control--horizontal--default) + var(--pf-v6-c-text-input-group__icon--FontSize) + var(--pf-t--global--spacer--gap--text-to-element--default));
-  --#{$text-input-group}--status__text-input--PaddingInlineEnd: var(--pf-t--global--spacer--xl);
+  --#{$text-input-group}--status__text-input--PaddingInlineEnd: calc(var(--pf-t--global--spacer--control--horizontal--default) + var(--pf-v6-c-text-input-group__icon--FontSize) + var(--pf-t--global--spacer--gap--text-to-element--default));
```

Comment:
This is awesome. But when the close button is present, we adjust the status button's inset from `--pf-t--global--spacer--control--horizontal--default` to `--pf-t--global--spacer--sm` with the other change in this PR. That leaves a little too much padding between the status icon and text in the text input.

<img width="865" alt="Screenshot 2024-08-12 at 9 34 31 PM" src="https://github.com/user-attachments/assets/84e28f41-4dc5-47cc-a327-00121f60ddc9">

WDYT about creating a variation of this var that swaps `--pf-t--global--spacer--control--horizontal--default` with `--pf-t--global--spacer--sm`? Then it looks like this, with the visual spacer being the text-to-element spacer:

<img width="922" alt="Screenshot 2024-08-12 at 9 31 54 PM" src="https://github.com/user-attachments/assets/eb8d523b-3151-43f5-891b-26c6f2c1edfd">

Reply:
Pushed, lemme know if that's what you had in mind. FWIW updating the existing var so that it starts as `calc(var(--#{$text-input-group}__icon--m-status--InsetInlineEnd)...` also worked, though I imagine if a consumer is updating that specific var we wouldn't want that to affect the spacing between status icon and text input.